packages/changed-elements-react/src/utils/utils.ts

```diff
@@ -69,3 +69,14 @@ export async function tryXTimes<T>(func: () => Promise<T>, attempts: number, del
   }
   throw error;
 }
+
+/** Creates map from an array of values.
+ * Expects createKey to supply a unique key per entry; otherwise will cause other entries with same key to me overwritten.
+*/
+export const arrayToMap = <T, U>(array: T[], createKey: (entry: T) => U) => {
```

Comment:
This function is borderline useless because one can already write equivalent code in the following way:

```ts
new Map(array.map((entry) => [createKey(entry), entry]));
```

Reply:
I guess you could make an argument to keep the function if it was used in many more places.